apis/vertexai/v1alpha1/types.generated.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	refsv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+)
+
+// +kcc:proto=google.cloud.aiplatform.v1beta1.EncryptionSpec
+type EncryptionSpec struct {
+	// Required. The Cloud KMS resource identifier of the customer managed
+	//  encryption key used to protect a resource. Has the form:
+	//  `projects/my-project/locations/my-region/keyRings/my-kr/cryptoKeys/my-key`.
+	//  The key needs to be in the same region as where the compute resource is
+	//  created.
+	// +kcc:proto:field=google.cloud.aiplatform.v1beta1.EncryptionSpec.kms_key_ref
+	KMSKeyRef *refsv1beta1.KMSCryptoKeyRef `json:"kmsKeyRef,omitempty"`
+}
+
+// +kcc:proto=google.cloud.aiplatform.v1beta1.MetadataStore
+type MetadataStore struct {
+
+	// Customer-managed encryption key spec for a Metadata Store. If set, this
```

Comment:
Can we use the generated code here for spec as well? I am seeing the same fields in `VertexAIMetadataStoreSpec` but they are not properly annotated and the field comments do not match. Not sure if there is a special reason to not use these?